src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>You could slightly tweak this if you want:</p><p> </p><p>-&gt; Takeaways here include:</p><ul><li>the data is right-skewed</li><li>there is a positive correlation between <code>trip_distance</code> and <code>trip_duration</code></li><li>most trips are completed in under 3600 seconds (one hour)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>